kurl_util/cmd/yamltobash/main.go

```diff
@@ -195,6 +195,7 @@ func convertToBash(kurlValues map[string]interface{}, fieldsSet map[string]bool)
 		"Kotsadm.ApplicationSlug":                "KOTSADM_APPLICATION_SLUG",
 		"Kotsadm.Hostname":                       "KOTSADM_HOSTNAME",
 		"Kotsadm.S3Override":                     "KOTSADM_S3_OVERRIDE",
+		"Kotsadm.MigrateS3":                      "KOTSADM_MIGRATE_S3",
```

Comment:
need to find a better name for this

Reply:
`NO_OBJECT_STORE`?
`EXCLUDE_OBJECT_STORE`?
`STATEFUL_KOTSADM`?